scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -87,7 +87,6 @@ check-runtime-migration-polkadot:
     - .check-runtime-migration
   variables:
     NETWORK: "polkadot"
-  allow_failure: true # FIXME https://github.com/paritytech/substrate/issues/14006
```

Comment:
I think we need to keep all of them allow-failure, or create another job that runs without the `pre-and-post` checks.  
That could then be required to go green.